db/migrate/20221005095551_create_tickets.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTickets < ActiveRecord::Migration[7.0]
+  def change
+    create_table :tickets do |t|
+      t.integer :seat, null: false
```

Comment:
perhaps it would be better to have this column in `string` type? You don't have to change the type when a client says "now we will support halls with seats like A1, A2 and so on" :D

Reply:
My validations right now are based on seats being and integer, i think it's really good point, i'm gonna try to change things to allow string but probably after deadline